mapbook-iOS/AppDelegate.swift

```diff
@@ -30,16 +30,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
-        
-        if let urlComponents = URLComponents(url: url, resolvingAgainstBaseURL: false),
-            urlComponents.scheme == AppSettings.appSchema, urlComponents.host == AppSettings.authURLPath {
-            
-            AGSApplicationDelegate.shared().application(app, open: url, options: options)
```

Comment:
This method is deprecated in U11.

> "The 'ASWebAuthenticationSession' is used instead of 'SFSafariViewController' for out-of-app authentication workflow which does not require this delegate method."